opencti-platform/opencti-front/src/components/ItemAccountStatus.tsx

```diff
@@ -80,7 +85,12 @@ const ItemAccountStatus: FunctionComponent<ItemAccountStatusProps> = ({
       classes={{ root: style }}
       variant={variant}
       label={label}
-      style={classStyle}
+      style={{
+        ...classStyle,
+        color: theme.palette.chip.main,
+        borderColor: monochrome_labels ? theme.palette.background.accent : classStyle.borderColor,
```

Comment:
Just having a thought right now : wouldn't it be better to have a attribute in the chip if we want the monochrome or the normal behavior and have the theme.palette.... inside all chips directly ?